src/org/loklak/harvester/strategy/KaizenQueries.java

```diff
@@ -26,7 +30,7 @@ public static KaizenQueries getDefaultKaizenQueries(int qLimit) {
             int queryLimit = qLimit;
 
             @Override
-            public boolean addQuery(String query) {
+            public boolean addQuery(String query, double score) {
```

Comment:
Because for any type of `KaizenQueries`, the `private` methods of `KaizenHarvester` that signal the addition of new queries will be the same. i.e., it is not possible to modify the add call if the type of `KaizenQueries` is different. So, it is passed to all type of `KaizenQueries`' `addQuery` method, and it is up to them to use it.